### PR TITLE
doc: enrich using-gc-traces

### PR DESCRIPTION
**Pull request by tony-go**
Hi 👋 

## Context

I recently wrote down a [guide](https://github.com/nodejs/nodejs.org/pull/4590/files) and pushed a PR on this repo. 

As discussed in the recent meeting, it was probably not the best approach. We thought that it could be better to merge it with existing documentation.

## Update

But we finally decided to extract a few parts from my work and inject it into the existing doc.

> Note: We should merge this pull request first https://github.com/nodejs/diagnostics/pull/572, as it will expose links we need in this pr.

### CODE REVIEW

**Comment by tony-go** [comment]
Cc @nodejs/diagnostics

File: locale/en/docs/guides/diagnostics/memory/using-gc-traces.md [hunk 1]
```diff
@@ -5,99 +5,321 @@ layout: docs.hbs
...
+is where objects that survived the new space are living.
+
+This algorithm is composed of two phases:
+* **Mark**: Will mark still alive objects as black and others as white.
+* **Sweep**: Scans for white objects and converts them to free spaces.
+
+> 👉 In fact, the Mark and Sweep steps are a bit more elaborate.
+> Please read this [document][] for more details.
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/4/4a/Animation_of_the_Naive_Mark_and_Sweep_Garbage_Collector_Algorithm.gif" alt="mark and sweep algorithm"/>
```
Comment (ovflowd) [inline locale/en/docs/guides/diagnostics/memory/using-gc-traces.md:166]: Is this image in any sort of CC licensing? Are we allowed to use it?

**Review by ovflowd — APPROVED** [review]
Very nice exercise! Concise and very explanatory. From the website side of things, I don't see anything wrong. Still need a +1 review from diagnostics 😄

**Review by RafaelGSS — APPROVED** [review]
LGTM.

File: locale/en/docs/guides/diagnostics/memory/using-gc-traces.md (hunk 1 above)
Comment (tony-go) [reply locale/en/docs/guides/diagnostics/memory/using-gc-traces.md:166]: I think that yes. Just found this: https://en.wikipedia.org/wiki/Wikipedia:Ten_things_you_may_not_know_about_images_on_Wikipedia

Comment (ovflowd) [reply locale/en/docs/guides/diagnostics/memory/using-gc-traces.md:166]: Nice! Thanks for answering the question, @tony-go! We're good to go then 🚀

File: locale/en/docs/guides/diagnostics/memory/using-gc-traces.md [hunk 2]
```diff
@@ -146,78 +368,25 @@ PerformanceEntry {
...
-  </tr>
-</table>
+| Property  | Interpretation                                                                                   |
+|-----------|--------------------------------------------------------------------------------------------------|
+| name      | The name of the performance entry.                                                               |
+| entryType | The type of the performance entry.                                                               |
+| startTime | The high-resolution millisecond timestamp is marking the starting time of the Performance Entry. |
+| duration  | The total number of milliseconds elapsed for this entry.                                         |
+| kind      | The type of garbage collection operation that occurred.                                          |
+| flags     | The high-resolution millisecond timestamp is marking the starting time of the Performance Entry. |
```
Comment (ghost) [inline locale/en/docs/guides/diagnostics/memory/using-gc-traces.md:378]: "flags"may be the meaning of "The additional info about GC"?

Comment (MaledongGit) [inline locale/en/docs/guides/diagnostics/memory/using-gc-traces.md:378]: "flags"may be the meaning of "The additional info about GC"?

**Review by ghost — COMMENTED** [review]
@tony-go："flags"may be the meaning of "The additional info about GC"?

**Comment by ghost** [comment]
@tony-go: Sorry but catch it a bit later.....:(

**Comment by tony-go** [comment]
> @tony-go: Sorry but catch it a bit later.....:(

Nice catch, I probably miss something when I moved blocks in the doc.

**Pull request merged**